prac_07/distance_converter.py

```diff
@@ -0,0 +1,26 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.app import StringProperty
+
+
+class DistanceConverter(App):
+    message = StringProperty()
+
+    def build(self):
+        self.title = 'Convert Miles to Kilometres'
+        self.root = Builder.load_file('distance_converter.kv')
+        return self.root
+
+    def handle_calculate(self, value):
+        result = value * 1.60934
```

Comment:
Any time you use a literal (like 1.6...) multiple times, it should probably be a constant.